app/core/tests/test_api.py

```diff
@@ -156,3 +156,17 @@ def test_create_sponsor_partner(auth_client):
     }
     res = auth_client.post(SPONSOR_PARTNERS_URL, payload)
     assert res.status_code == status.HTTP_201_CREATED
+
+
+def test_create_faq(admin_client):
```

Comment:
It's better to use auth_client. Admin_client was for testing permissions, but it's not used anymore.